<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExcepEditor</class>
 <widget class="QWidget" name="ExcepEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>676</width>
    <height>563</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 9pt &quot;微软雅黑&quot;;</string>
  </property>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>631</width>
     <height>531</height>
    </rect>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAsNeeded</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAsNeeded</enum>
   </property>
   <property name="widgetResizable">
    <bool>false</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>962</width>
      <height>972</height>
     </rect>
    </property>
    <widget class="QPlainTextEdit" name="absEdit">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>60</y>
       <width>501</width>
       <height>81</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>标题</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="titleEdit">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>30</y>
       <width>491</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>摘要</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>850</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>处理人ID</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>920</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>时间（由系统自动获取）</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>详细描述</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>600</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>处理方案</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>360</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>发生原因</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>880</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>处理人姓名</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="detailEdit">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>150</y>
       <width>501</width>
       <height>161</height>
      </rect>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="reasonEdit">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>380</y>
       <width>501</width>
       <height>211</height>
      </rect>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="schemeEdit">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>620</y>
       <width>501</width>
       <height>171</height>
      </rect>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAsNeeded</enum>
     </property>
     <property name="tabChangesFocus">
      <bool>false</bool>
     </property>
     <property name="lineWrapMode">
      <enum>QPlainTextEdit::WidgetWidth</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="handlerIDEdit">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>850</y>
       <width>471</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="handlerNameEdit">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>880</y>
       <width>461</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="cancleButton">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>930</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>取消</string>
     </property>
    </widget>
    <widget class="QPushButton" name="handinButton">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>910</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>提交</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
